lib/line/bot/client.rb

```diff
@@ -61,6 +62,46 @@ def credentials?
         credentials.values.all?
       end
 
+      # Issue channel access token
+      #
+      # @param grant_type [String] Grant type
+      #
+      # @return [Net::HTTPResponse]
+      def issue_access_token(grant_type)
+        payload = URI.encode_www_form(
+          grant_type:    grant_type,
+          client_id:     channel_token,
```

Comment:
`client_id` is different from `channel_token`.
`client_id` is integer value, but `channel_token` is very long base64 encoded value.

So this method has to receive additional parameter `client_id`.

Reply:
Also I'm wondering we should receive `grant_type` as parameter, since currently it is always `client_credentials`...

Reply:
> client_id is different from channel_token.

Oh...  I see the document again.  Thanks!

Reply:
> Also I'm wondering we should receive grant_type as parameter, since currently it is always client_credentials...

Yeah, I considered about it too.  How about setting default value with argument, like:

```rb
def issue_access_token(grant_type = 'clinet_credentials')
```

Reply:
@toduq  For now, fixed with using default value for arguments.  If you have any idea, let me to share it, thanks!